bundles/go/init.moon

```diff
@@ -0,0 +1,52 @@
+import app, command, config, mode, signal from howl
+{:fmt} = bundle_load 'go_fmt'
+
+register_mode = ->
+  mode_reg =
+    name: 'go'
+    aliases: 'golang'
+    extensions: 'go'
+    create: -> bundle_load('go_mode')
+    parent: 'curly_mode'
+  
+  mode.register mode_reg
+  
+register_commands = ->
+  command.register
+    name: 'go-fmt',
+    description: 'Run go fmt on the current buffer and reload if reformatted'
+    handler: ->
+      fmt app.editor.buffer
+      
+buffer_saved = (args) ->
+  fmt args.buffer
+  
+register_mode!
+register_commands!
+signal.connect 'buffer-saved', buffer_saved
+
+with config
+  .define
+    name: 'go_fmt_command'
+    description: 'Command to run for go-fmt'
+    default: 'gofmt -w'
+    scope: 'global'
+
+  .define
+    name: 'go_complete'
+    description: 'Whether to use go completions in go mode'
```

Comment:
Maybe then 'based on gocode'? I'm assuming there are multiple completion engines out there and having this info in the description would be useful to anyone just browsing the config.